config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml

```diff
@@ -721,6 +729,7 @@ postsubmits:
   kubernetes/kubernetes:
   - annotations:
       testgrid-dashboards: google-unit, sig-release-job-config-errors
+      testgrid-alert-email: release-managers+alerts@kubernetes.io
```

Comment:
Do you think we should also add `release-team@kubernetes.io` to these bazel builds?

Reply:
we can add, I don't have a strong opinion on that